### ISSUE TITLE
Unable to build wheel for texture_baker

### ISSUE THREAD

**Issue by vantemoon** [Stability-AI/stable-fast-3d#55]
Hi, I'm unable to build the wheel for texture_baker. Even though I've confirmed that I have Visual Studio 2022 correctly installed, it still shows that my Visual Studio version is incompatible.

Python 3.11.10
Visual Studio Compiler 19.41.34123
Windows 11 x64
Torch 2.5.0 + cu121
CUDA 12.1

```
C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.1\include\crt/host_config.h(153): fatal error C1189: #error:  -- unsupported Microsoft Visual Studio version! Only the versions between 2017 and 2022 (inclusive) are supported! The nvcc flag '-allow-unsupported-compiler' can be used to override this version check; however, using an unsupported host compiler may cause compilation failure or incorrect run time execution. Use at your own risk.
      baker_kernel.cu
      error: command 'C:\\Program Files\\NVIDIA GPU Computing Toolkit\\CUDA\\v12.1\\bin\\nvcc.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for texture_baker
```

**Comment by vantemoon** [comment]
Solved by upgrading CUDA to 12.4, seems like the current version of Visual Studio does not support CUDA 12.1

**Comment by Jillinger** [comment]
I'm using CUDA to 12.4, with Python 3.13 and torch 2.6.0+cu124

```
pip install -r requirements.txt
Ignoring rembg: markers 'sys_platform == "darwin"' don't match your environment
Processing f:\z\texture_baker
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [23 lines of output]
      Traceback (most recent call last):
        File "F:\...\PythonVEnv\AI_GeneratorS\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 389, in <module>
          main()
          ~~~~^^
        File "F:\...\PythonVEnv\AI_GeneratorS\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 373, in main
          json_out["return_val"] = hook(**hook_input["kwargs"])
                                   ~~~~^^^^^^^^^^^^^^^^^^^^^^^^
        File "F:\...\PythonVEnv\AI_GeneratorS\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 143, in get_requires_for_build_wheel
          return hook(config_settings)
        File "C:\...\AppData\Local\Temp\pip-build-env-5g6pgtgd\overlay\Lib\site-packages\setuptools\build_meta.py", line 331, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=[])
                 ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\...\AppData\Local\Temp\pip-build-env-5g6pgtgd\overlay\Lib\site-packages\setuptools\build_meta.py", line 301, in _get_build_requires
          self.run_setup()
          ~~~~~~~~~~~~~~^^
        File "C:\...\AppData\Local\Temp\pip-build-env-5g6pgtgd\overlay\Lib\site-packages\setuptools\build_meta.py", line 512, in run_setup
          super().run_setup(setup_script=setup_script)
          ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\...\AppData\Local\Temp\pip-build-env-5g6pgtgd\overlay\Lib\site-packages\setuptools\build_meta.py", line 317, in run_setup
          exec(code, locals())
          ~~~~^^^^^^^^^^^^^^^^
        File "<string>", line 5, in <module>
      ModuleNotFoundError: No module named 'torch'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
ERROR: Failed to build 'file:///F:/Z/texture_baker' when getting requirements to build wheel
```
